<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPesqCliente</class>
 <widget class="QDialog" name="frmPesqCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista de Cliente</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.508, y1:0.727, x2:0.505, y2:1, stop:0 rgba(255, 255, 153, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="title">
    <string>Selecione o Tipo de Pesquisa</string>
   </property>
   <widget class="QLineEdit" name="edtPesquisa">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPesquisa">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Imagens/Pesquisar.png</normaloff>../../Imagens/Pesquisar.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPesquisa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <item>
     <property name="text">
      <string>Código</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nome</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CPF</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QTableWidget" name="gridCliente">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>621</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <column>
    <property name="text">
     <string>Código</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nome</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CPF</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Endereço</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Telefone</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnAlterar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
   </property>
   <property name="text">
    <string>Alterar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Imagens/Alterar.png</normaloff>../../Imagens/Alterar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExcluir">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>420</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
   </property>
   <property name="text">
    <string>Excluir</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Imagens/lixeira.png</normaloff>../../Imagens/lixeira.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
